src/GraphQL.Tests/Types/Relay/CursorTests.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using GraphQL.Types.Relay;
+using Xunit;
+
+namespace GraphQL.Tests.Types.Relay
+{
+    public class CursorTests
+    {
+        [Fact]
+        public void GetFirstAndLastCursor_NullCollection_ReturnsNull()
+        {
+            List<Entity> items = null;
+
+            var (firstCursor, lastCursor) = Cursor.GetFirstAndLastCursor(items, x => x.Property);
+
+            Assert.Null(firstCursor);
+            Assert.Null(lastCursor);
```

Comment:
This project uses Shoudly for assertions. Can you please update these to use that?

Reply:
Switched to Shoudly as requested.